### DISCUSSION TITLE
你好站长，账号不知道为什么被封了麻烦帮忙确认下原因

### DISCUSSION THREAD

**Discussion by invokerw** [BurnHair/docs#121] [技术支持]
{
    "message": "status code 403: 用户8498已被封禁：NSFW (request id: 20240625070119280438630cfudXDgC)",
    "success": false,
    "time": 0.32
}

**Comment by BennyThink** [comment]
<img width="841" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/59b55ee4-f67c-4782-aec2-0498299125c9">

😂虽然但是……

**Reply by invokerw** [reply]
好的23333
